backend/src/tasks/notifications.ts

```diff
@@ -0,0 +1,59 @@
+import { Handler } from 'aws-lambda';
+import { sendEmail } from '../api/helpers';
+import { connectToDatabase, Organization, Role } from '../models';
+import S3Client from './s3-client';
+
+const sendReportEmail = async (email: string, organization?: Organization) => {
+  await sendEmail(
+    email,
+    'New Report Available',
+    `Hi there,
+
+The latest report for ${
+      organization?.name ? `is now available on ` : ''
+    }Crossfeed. To view the report, sign on at ${process.env.FRONTEND_DOMAIN}
+
+Crossfeed access instructions:
+
+For more information on using Crossfeed, view the Crossfeed user guide at https://docs.crossfeed.cyber.dhs.gov/user-guide/quickstart/. 
+
+If you encounter any difficulties, please feel free to reply to this email (or send an email to ${
+      process.env.CROSSFEED_SUPPORT_EMAIL_REPLYTO
+    }).`
+  );
+};
+
+export const handler: Handler = async (event) => {
+  await connectToDatabase();
+  console.log('Running notifications check...');
+
+  // Get all organizations
+  const organizations = await Organization.find();
```

Comment:
Can we just filter CISA for now

in a later PR we can toggle notifications on / off for orgs

Reply:
updated in [4645cd5](https://github.com/cisagov/crossfeed/pull/1709/commits/4645cd5885e5a40120e0a11ba94d5fddc232a3a7)